src/components/dashboard/Dashboard.js

```diff
@@ -119,7 +120,7 @@ const Dashboard = props => {
       await executeWithdraw(store, paymentCode, reason)
       hideDialog()
     } catch (e) {
-      showDialog({ title: 'Error', message: e.message })
+      showErrorDialog(e)
```

Comment:
This error dialog is displaying like a normal dialog...

![image](https://user-images.githubusercontent.com/3315606/62311790-fa42fe80-b462-11e9-8d1e-46099aa354e5.png)

It's not part of this PR, but we can take the opportunity to fix it in here.

Reply:
good catch! fixed dialogs implementation